backoffice/src/test/java/com/wedriveu/backoffice/analytics/AnalyticsVehicleCounterRequestReceiverConsumer.java

```diff
@@ -0,0 +1,47 @@
+package com.wedriveu.backoffice.analytics;
+
+import com.wedriveu.backoffice.util.ConstantsBackoffice;
+import com.wedriveu.services.shared.rabbitmq.VerticleConsumer;
+import com.wedriveu.shared.util.Constants;
+import io.vertx.core.Future;
+import io.vertx.core.eventbus.Message;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * @author Stefano Bernagozzi
+ */
+public class AnalyticsVehicleCounterRequestReceiverConsumer extends VerticleConsumer {
```

Comment:
It's a loooong name, why not `AnalyticsVehicleCounterConsumer` or `AnalyticsVehicleCounterRequestConsumer`?